p2p/peer.py

```diff
@@ -169,6 +168,7 @@ def __init__(self,
         self.headerdb = headerdb
         self.network_id = network_id
         self.inbound = inbound
+        self._subscribers: List['asyncio.Queue[PEER_MSG_TYPE]'] = []
```

Comment:
If you define `PEER_MSG_TYPE = Tuple['BasePeer', protocol.Command, protocol._DecodedMsgType]`

At the beginning of the file, you can get rid of all the string type references (aka forward refs)

You can see this (or even pick from) here: https://github.com/cburgdorf/py-evm/commit/f5d534dfe062b5ebc140c54ac4462d6cf727de19

Reply:
I cannot, because `asyncio.Queue` is an actual type and python will give a type error (TypeError: 'type' object is not subscriptable)

Reply:
Oh, you're right. Never mind.